extension-androidauto/src/main/java/com/mapbox/maps/extension/androidauto/MapboxCarUtils.kt

```diff
@@ -0,0 +1,180 @@
+package com.mapbox.maps.extension.androidauto
+
+import android.graphics.Rect
+import android.util.Log
+import androidx.car.app.AppManager
+import androidx.car.app.Session
+import androidx.car.app.SurfaceCallback
+import androidx.car.app.SurfaceContainer
+import androidx.lifecycle.DefaultLifecycleObserver
+import androidx.lifecycle.LifecycleOwner
+import com.mapbox.common.Logger
+import com.mapbox.maps.*
+
+/**
+ * MapSurface ready callback, will be called when the MapSurface is created successfully.
+ */
+@MapboxExperimental
```

Comment:
Should we even mark everything as Mapbox experimental? 
I guess it's more of Google experimental as APIs could easily change for Android Auto

Reply:
These just notify the users that this feature is unstable and depends on an alpha version of Android Car App library. We may change these interfaces if Google decide to change their API surface.